### PR TITLE
Replace rebase with merge

### PR DESCRIPTION
**Pull request by artasparks**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -168,18 +164,14 @@ First, here's what my workflow looks like (which is quite similar to this
     ```shell
     git checkout master
     git fetch upstream
-    git rebase -i upstream master
+    git merge -i upstream master
```
Comment (nelsonjhk) [inline CONTRIBUTING.md]: '-i' flag should be removed? Also add slash: "upstream/master".

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -168,18 +164,14 @@ First, here's what my workflow looks like (which is quite similar to this
...
     git push
 
     # Update feature branch
     git checkout somefeature
-    git rebase -i master
+    git merge -i master
```
Comment (nelsonjhk) [inline CONTRIBUTING.md]: '-i' flag should be removed?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -146,7 +142,7 @@ First, here's what my workflow looks like (which is quite similar to this
     ```shell
     git checkout master
     git fetch upstream
-    git rebase -i upstream master
+    git merge -i upstream master
```
Comment (nelsonjhk) [inline CONTRIBUTING.md]: '-i' flag should be removed? Also add slash: "upstream/master".

**Review by nelsonjhk — CHANGES_REQUESTED** [review]
Noted some typos.

File: CONTRIBUTING.md (hunk 1 above)
Comment (artasparks) [reply CONTRIBUTING.md]: Done

File: CONTRIBUTING.md (hunk 2 above)
Comment (artasparks) [reply CONTRIBUTING.md]: Done

File: CONTRIBUTING.md (hunk 3 above)
Comment (artasparks) [reply CONTRIBUTING.md]: Done

**Review by nelsonjhk — APPROVED** [review]
lgtm

**Review by nelsonjhk — APPROVED** [review]
lgtm

**Pull request merged**